spec/model_spec.cr

```diff
@@ -97,4 +120,48 @@ describe LuckyRecord::Model do
       QueryMe.column_names.should eq [:id, :created_at, :updated_at, :email, :age]
     end
   end
+
+  describe ".ensure_correct_field_mappings" do
+    # table is missing
+    it "raises on missing table" do
+      missing_table = MissingTable.new(1, Time.new, Time.new)
+      expect_raises Exception, "The table uusers was not found, did you mean users?" do
+        missing_table.ensure_correct_field_mappings!
+      end
+    end
+
+    # field defined in model without a matching column in table
+    it "raises on fields with missing columns" do
+      now = Time.now
+      user = MissingColumn.new id: 1,
+        created_at: now,
+        updated_at: now,
+        missing: "missing"
+      expect_raises Exception, "The table users does not have a 'missing' column. Make sure you've added it to the migration." do
```

Comment:
Its not relevant to which migration you actually add a field, is it? The `to the migration` I find slightly confusing. To which migration? Maybe `added it in one of your apps migrations` or something?

Reply:
Sure, how about `Make sure you've added it to a migration`

Reply:
I like that 👍